### PR TITLE
[ISSUE #3101] Added final modifier for lines 34,36,38,44

### LINKED ISSUE #3101: [Enhancement] Field may be 'final'[GrpcRetryer]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
image
located at:
eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/retry/GrpcRetryer.java line 35,37,39,45,65

### Describe the solution you'd like

Add the 'final' modifier.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by PickBas**
Closes #3101

### Modifications

Added `final` modifier for `GrpcRetryer` fields. 

### Documentation

- Does this pull request introduce a new feature? - no
- If yes, how is the feature documented? - not applicable 
- If a feature is not applicable for documentation, explain why? - minor enhancement

### CODE REVIEW

**Review by mxsm — DISMISSED** [review]

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/retry/GrpcRetryer.java [hunk 1]
```diff
@@ -31,17 +31,17 @@
 
 public class GrpcRetryer {
 
-    private Logger retryLogger = LoggerFactory.getLogger("retry");
+    private final Logger retryLogger = LoggerFactory.getLogger("retry");
```
Comment (jonyangx) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/retry/GrpcRetryer.java:34]: Suggest merge loggers using lombok sl4j annotation.

**Comment by PickBas** [comment]
@jonyangx Please, review

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/retry/GrpcRetryer.java [hunk 2]
```diff
@@ -29,27 +30,24 @@
 import org.slf4j.Logger;
```
Comment (jonyangx) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/retry/GrpcRetryer.java:30]: Suggest remove the redundant code.

Comment (PickBas) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/grpc/retry/GrpcRetryer.java]: Done! @jonyangx

**Review by jonyangx — APPROVED** [review]

**Comment by jonyangx** [comment]
@PickBas pls fix build error.

**Comment by PickBas** [comment]
@jonyangx Take a look

**Review by jonyangx — APPROVED** [review]

**Comment by HattoriHenzo** [comment]
@PickBas seems like you have made your modifications directly in the **master** branch. I will suggest to repost your changes in a new branch. https://eventmesh.apache.org/community/contribute/contribute

**Pull request merged**